src/sentry/tagstore/snuba/backend.py

```diff
@@ -16,69 +17,363 @@
 from sentry.tagstore.base import TagStorage
 from sentry.utils import snuba
 
+SEEN_COLUMN = 'received'
+
+
+class ObjectWrapper(object):
+    def __init__(self, dictionary):
+        self.__dict__ = dictionary
+
 
 class SnubaTagStorage(TagStorage):
 
+    def get_time_range(self, days=90):
```

Comment:
We probably want to note a TODO to use something like this: https://github.com/getsentry/sentry/blob/9c87c1ae59f6d1eae39e2bcecee0b192cb8bf3f5/src/sentry/search/django/backend.py#L241-L251

Reply:
done